src/openzaak/conf/dev.py

```diff
@@ -18,6 +18,8 @@
 os.environ.setdefault("ENVIRONMENT", "development")
 os.environ.setdefault("DISABLE_2FA", "True")
 os.environ.setdefault("LOG_FORMAT_CONSOLE", "plain_console")
+os.environ.setdefault("USE_STRUCTLOG", "True")
```

Comment:
Let's move this line to `base.py` before the import from `open_api_framework.conf.base` happens and modify it, that way it's always enabled

```suggestion
os.environ["USE_STRUCTLOG"] = "True"
```